adam/language_specific/english/english_language_generator.py

```diff
@@ -142,10 +168,22 @@ def _translate_object_to_noun(
             if (dependency_node.part_of_speech != PROPER_NOUN) and (
                 MASS_NOUN not in lexicon_entry.properties
             ):
-                determiner_node = DependencyTreeToken("a", DETERMINER)
-                self.dependency_graph.add_edge(
-                    determiner_node, dependency_node, role=DETERMINER_ROLE
-                )
+                if count == 1:
+                    determiner_node = DependencyTreeToken("a", DETERMINER)
+                    self.dependency_graph.add_edge(
+                        determiner_node, dependency_node, role=DETERMINER_ROLE
+                    )
+                elif count == 2:
+                    numeral_node = DependencyTreeToken("two", NUMERAL)
+                    self.dependency_graph.add_edge(
+                        numeral_node, dependency_node, role=NUMERIC_MODIFIER
+                    )
+                # Currently, any number of objects greater than two is considered "many"
+                else:
+                    adjective_node = DependencyTreeToken("many", ADJECTIVE)
```

Comment:
Does universal dependencies analyze this as `ADJECTIVE` or `DETERMINER`?

Reply:
By definition it seems like it could be a determiner, but CoreNLP tags it as an adjective. Is there a better way to confirm this?

Reply:
Changed POS to `DETERMINER`